docs/blog/2019-06-21-profitable-agency-wordpress/index.md

```diff
@@ -0,0 +1,115 @@
+
+I once discovered a bug caused by a page builder plugin update that prevented a theme from displaying comments. I spent half a day sussing out the culprit, then another day getting the page builder's support team to take responsibility and recognize it as a true bug. Then a week later it was fixed.
+
+Because WordPress plugins are writing to the same database, using the same platform functions, and jockeying for space in the same admin area, the permutations of potential problems is high.  And solving issues can take a lot of time.  Also, unless you lock down your WordPress projects with as little user privileges as possible, then you're sure to get a call every so often about something your client broke themself.
+
+Gatsby is different. Consider this example:
+
+If your client needs ecommerce features on their site, Gatsby can pull in the product data and shopping cart functionality from two different services, if you so choose. Those two services are not produced by the same server, or writing to the same database. So a problem with one may not affect the other.
+
+In theory, break-fix for Gatsby should be easier to diagnose and solve. An issue with the build has nothing to do with a problem with the front-end, and vice versa. Gatsby's lower technical maintenance may be the difference between you taking on 5 client sites or 15.
+
+
+> Personally I keep forgetting about the static sites under my care because there's literally nothing to work on for them month-to-month.
+
+## Freemium is cheaper than pay-per-year Plugins
+
+The WordPress ecosystem is bursting with solutions, paid and free, for any use case your clients have. I'm personally very grateful for that.
+
+Recently, premium plugin companies have discovered that the value of their software, and the cost of customer service, are both higher than expected. If you think break-fix is hard for us, imagine having your code run on thousands of different environments, and hosting platforms, and versions of PHP. So, of course, plugin prices have gone up. I don't begrudge the fact that many of my favorite WordPress shops are asking for more.
+
+Again, the Gatsby ecosystem is different. Like hosting, the standard pricing for third-party website services is a generous free tier.
+
+There are a dozen [Gatsby-friendly form builders](https://www.gatsbyjs.org/docs/building-a-contact-form) with unlimited trials and free tiers. And if you create a form handler yourself, using a React library for forms, plus [Amazon API Gateway / Lambda / SES](https://aws.amazon.com/blogs/architecture/create-dynamic-contact-forms-for-s3-static-websites-using-aws-lambda-amazon-api-gateway-and-amazon-ses/), then your client can handle half a million submissions a month for under $5.
```

Comment:
```suggestion
There are a dozen [Gatsby-friendly form builders](/docs/building-a-contact-form) with unlimited trials and free tiers. And if you create a form handler yourself, using a React library for forms, plus [Amazon API Gateway / Lambda / SES](https://aws.amazon.com/blogs/architecture/create-dynamic-contact-forms-for-s3-static-websites-using-aws-lambda-amazon-api-gateway-and-amazon-ses/), then your client can handle half a million submissions a month for under $5.
```
One tweak and this will be good to go!